.github/workflows/workspace-ci-unsafe.yml

```
@@ -23,6 +24,11 @@ jobs:
     strategy:
       matrix: ${{ fromJson(needs.setup.outputs.matrix) }}
     uses: ./.github/workflows/reusable-should-validate.yml
+    secrets:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.7)

secret "GITHUB_TOKEN" is not defined in "./.github/workflows/reusable-should-validate.yml" reusable workflow. no secret is defined

(workflow-call)